### DISCUSSION TITLE
Demarking different layers

### DISCUSSION THREAD

**Discussion by arnab-pal** [emanuelhuber/RGPR#76] [Q&A]
I have already performed all the processing steps in RGPR. I want to know the steps of identifying the layers.

**Comment by emanuelhuber** [comment]
Hello @arnab-pal 

Thank you for your interest in RGPR. What do you mean with "steps of identifying the layers"? Do you mean:

- Interprete the signal reflections and relate them to some structures of the investigated media?
- Delineate some reflections? Like drawing lines on your plot?

Best,

Emanuel

**Comment by arnab-pal** [comment]
In different softwares, there is option of layer picking based on (phase/ manual/ point). Based on that layers present in the section can be identified. I am asking whether there is any option in RGPR or not.
Thanking you.
